Dockerfile.logger

```diff
@@ -3,6 +3,7 @@ FROM python:3.10
 WORKDIR /usr/app/src
 
 COPY logger.py ./
+COPY api ./api
```

Comment:
@OleksandrUA why are you adding this?

Reply:
@cristidas `logger.py` (expects)[https://github.com/makerdao/chainlog-ui/blob/main/logger.py#L74] this directory to be present and without this change it wasn't starting in K8s. It did start in ECS though, probably due to different runtime.

Reply:
@nickkounz I'd be fine with this but improvement of services is out of current scope of current initiative, we want to migrate services to K8s first and we can start planning the improvements of particular services later. At least this is what team aligned on planning the initiative